examples/cra/package.json

```diff
@@ -31,6 +34,15 @@
     ]
   },
   "devDependencies": {
+    "buffer": "npm:buffer@6.0.3",
+    "assert": "npm:assert@2.1.0",
+    "http": "npm:stream-http@^3.2.0",
+    "https": "npm:https-browserify@^1.0.0",
+    "zlib": "npm:browserify-zlib@^0.2.0",
+    "crypto": "npm:crypto-browserify@3.12.0",
+    "stream": "npm:stream-browserify@3.0.0",
+    "url": "npm:url@0.11.3",
+    "util": "npm:util@0.12.5",
```

Comment:
these are all the libs u mentioned we depend on? 
This is what you plan to improve? is the final outcome that we don't need to add any of these dev dependencies?

Reply:
Yes, I was talking about them.
But I've discovered that they actually came from `@safe-global/api-kit` and `@safe-global/protocol-kit`.
So, there is no problem with SDK itself.
I've tried to up Safe libs versions but haven't succeed, because they are using ethers@6.x.x.
In conclusion:
 - cow-sdk is good, no issues there
 - Safe libs with ethers@5.x.x have issues with node lbs
 - we can't update Safe libs because new versions are based on ethers@6.x.x

Reply:
ups!, I see

Reply:
Why is it prefixed with `npm:` ?